packages/cli/src/commands/debug-replay/debug-replay.command.ts

```diff
@@ -128,5 +117,6 @@ export const debugReplay = buildCommand("debug-simulation")
       string: true,
       description: "Path to cookies to inject before replay",
     },
+    ...COMMON_REPLAY_OPTIONS
```

Comment:
This adds a bunch of options which aren't actually used in the handler:

```
const handler: (options: Options) => Promise<void> = async ({
  apiToken,
  sessionId,
  appUrl,
  devTools,
  shiftTime,
  networkStubbing,
  moveBeforeClick,
  cookiesFile,
  disableRemoteFonts,
}) => {
```

```
export const COMMON_REPLAY_OPTIONS = {
  headless: OPTIONS.headless,
  devTools: OPTIONS.devTools,
  bypassCSP: OPTIONS.bypassCSP,
  padTime: OPTIONS.padTime,
  shiftTime: OPTIONS.shiftTime,
  networkStubbing: OPTIONS.networkStubbing,
  skipPauses: OPTIONS.skipPauses,
  disableRemoteFonts: OPTIONS.disableRemoteFonts,
};
```

It's probably a bit misleading to have them show in the CLI help but not actually be wired through.

Maybe for now do:

```
  devTools: OPTIONS.devTools,
  shiftTime: OPTIONS.shiftTime,
  networkStubbing: OPTIONS.networkStubbing,
  disableRemoteFonts: OPTIONS.disableRemoteFonts,
```

Can then always separately wire through more options.